web-app/CKS.Web.Test/IntegrationTests/Infrastructure/CKSWebApplicationFactory.cs

```diff
@@ -49,6 +53,12 @@ public CKSWebApplicationFactory WithImplementation<TService>(TService implementa
 			return this;
 		}
 
+		public CKSWebApplicationFactory SetStaticContentPath(string path)
```

Comment:
Can this be UseWebRoot to mimic the .NET method?

Reply:
A good suggestion as thats more generic.